package.json

```diff
@@ -4,8 +4,8 @@
     "description": "Exabyte Source of Schemas and Examples",
```

Comment:
We should advance the version of this package, like you did in the others

Reply:
✔️